conformance/chaos/actor.go

```diff
@@ -230,3 +234,21 @@ func (a Actor) MutateState(rt runtime.Runtime, args *MutateStateArgs) *adt.Empty
 	}
 	return nil
 }
+
+// AbortArgs are the arguments to the abort method, specifying the exit code to
+// (optionally) abort with and the message.
+type AbortArgs struct {
+	Code    exitcode.ExitCode
+	NoCode  bool
+	Message string
+}
+
+// Abort simply causes a panic or abort with the passed exit code.
+func (a Actor) Abort(rt runtime.Runtime, args *AbortArgs) *adt.EmptyValue {
```

Comment:
Mind calling it `AbortWith`? Just to distinguish it clearly from the many other `Abort` methods that exist.